physics/GFS_rrtmgp_pre.F90

```diff
@@ -164,7 +166,8 @@ subroutine GFS_rrtmgp_pre_run(me, nCol, nLev, nTracers, i_o3, lsswr, lslwr, fhsw
     integer, intent(out) :: &  
```

Comment:
Same comment RE: intent(out). Since many of these routines immediatley return if doSWrad and doLWrad are F, any intent(out) variables will automatically be set to random garbage upon return (whatever was previously in that memory space). This may not matter if they're not used, but it is good practice (and in the CCPP rules somewhere?) to initialize all intent(out) variables in CCPP schemes.